params/config_arbitrum.go

```diff
@@ -92,6 +96,17 @@ func ArbitrumRollupGoerliTestnetParams() ArbitrumChainParams {
 	}
 }
 
+func ArbitrumRinkebyTestParams() ArbitrumChainParams {
+	return ArbitrumChainParams{
+		EnableArbOS:               true,
+		AllowDebugPrecompiles:     false,
+		DataAvailabilityCommittee: false,
+		InitialArbOSVersion:       3,
+		// Not used - has init data
+		InitialChainOwner: common.Address{},
```

Comment:
Looking over the code again, I'm not sure we migrate chain owners, so this might be needed. I'd like to get this PR merged though, so I'll just make a follow-up PR to correct the comment.

Reply:
https://github.com/OffchainLabs/go-ethereum/pull/132